packages/registry/docs/howto/how-to-register-an-addon.md

```diff
@@ -32,3 +34,30 @@ export default function applyConfig(config: ConfigType) {
   return config
 };
 ```
+
+## Via a configuration file
+
+The configuration file can be provided via an environment variable.
+Because of legacy reasons, you can use one of these two:
+
+- `REGISTRYCONFIG`.
+- `VOLTOCONFIG`
+
```

Comment:
Fill in the blank, and adjust text and code example.

````suggestion

The value of the environment variable must point to a file that exists relative to the ______.

For example, if your configuration file is named {file}`my-add-on-config.json` and is located at the root of your add-on package, you would set your environment variable as shown.

```shell
set REGISTRYCONFIG="my-add-on-config.json"
```

````